src/Network/Network/Network.format.ps1xml

```diff
@@ -2104,10 +2104,14 @@
                 <Label>ProvisioningState</Label>
                 <PropertyName>ProvisioningState</PropertyName>
               </ListItem>
-              <ListItem>
-                <Label>Tags</Label>
-                <PropertyName>TagsTable</PropertyName>
-              </ListItem>
+			  <ListItem>
```

Comment:
Please use spaces instead of tabs to keep the consistency of indentation this file